manage_app.py

```diff
@@ -41,6 +34,29 @@ class bcolors:
     CEND = '\033[0m'
 
 
+args_parser = ArgumentParser(description="CLI args for the script")
+
+args_parser.add_argument(
+    '--load_env', action='store_true',
```

Comment:
Arguments are written with dashes instead of underscores. `--load-env`

Reply:
I see what you're doing with the underscores. Having the arguments as readable class members. Nevertheless, I think it's better if we stick to the proper naming convention.